indra/newview/CMakeLists.txt

```diff
@@ -1699,6 +1699,7 @@ if (WINDOWS)
         LINK_FLAGS_DEBUG "/NODEFAULTLIB:\"LIBCMT;LIBCMTD;MSVCRT\" /INCREMENTAL:NO /LARGEADDRESSAWARE"
         LINK_FLAGS_RELEASE "/FORCE:MULTIPLE /MAP\"secondlife-bin.MAP\" /OPT:REF /LARGEADDRESSAWARE"
         )
+    target_compile_options(${VIEWER_BINARY_NAME} PUBLIC /bigobj)
```

Comment:
I don't think it matters since we're not building a library target with downstream dependencies, but I would consider making this PRIVATE

Reply:
Makes sense. I'll change that.